assets/helpers/contributions.js

```diff
@@ -14,7 +14,8 @@ import type { Currency } from 'helpers/internationalisation/currency';
 
 // ----- Types ----- //
 
-export type Contrib = 'ANNUAL' | 'MONTHLY' | 'ONE_OFF';
+export type RegularContributionType = 'ANNUAL' | 'MONTHLY';
+export type Contrib = RegularContributionType | 'ONE_OFF';
```

Comment:
It would be nice to rename this to be `ContributionType`

Reply:
I know. That's a *big* refactor.